src/EditorFeatures/Core/Shared/Options/FeatureOnOffOptions.cs

```diff
@@ -86,6 +86,11 @@ internal static class FeatureOnOffOptions
 
         public static readonly PerLanguageOption2<bool> AddImportsOnPaste = new(
             nameof(FeatureOnOffOptions), nameof(AddImportsOnPaste), defaultValue: false);
+
+        public static readonly Option2<bool> OfferRemoveUnusedReferences = new(
+            nameof(FeatureOnOffOptions), nameof(OfferRemoveUnusedReferences), defaultValue: false,
+            storageLocations: new RoamingProfileStorageLocation($"TextEditor.{nameof(OfferRemoveUnusedReferences)}"));
```

Comment:
Is there some plan to also roll this out based on a feature flag?

Reply:
There wasn't, but I think that is a good idea for internal testing. Gonna get started on it. Thanks!